androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/EnhancedVideoPlayer.kt

```diff
@@ -122,20 +123,34 @@ fun EnhancedVideoPlayer(
             exoPlayer.removeListener(listener)
         }
     }
-
-    LaunchedEffect(isTimeBarDragged) {
-        if (isTimeBarDragged) {
-            exoPlayer.pause()
-        }
-    }
-
     TimeoutEffect(
         timeoutInMs = currentTimeTickInMs,
         enabled = isPlaying && isControlsVisible
     ) {
         currentTime = exoPlayer.currentPosition
     }
 
+    LaunchedEffect(isFullScreen) {
+        if (!isFullScreen) {
+            context.resetActivityBrightnessToDefault()
+            topDisplayCutout = context.getTopDisplayCutoutDp()
```

Comment:
note: i need to read the display cutout before go to the fullscreen mode, because when i try to read the display cutout in fullscreen the system returns 0 for the size of cutout. So i just save in portrait and remember the value.